recipes/kealib/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.4.6" %}
+
+package:
+    name: kealib
+    version: {{ version }}
+
+source:
+    fn: kealib-{{ version }}.tar.gz
+    url: https://bitbucket.org/chchrsc/kealib/downloads/kealib-{{ version }}.tar.gz
+    md5: 789174bd519736ac1e726613b6eb7672
+
+build:
+    number: 0
+    features:
+        - vc9  # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
+        
+requirements:
+    build:
+        - python  # [win]    
+        - cmake
+        - hdf5 1.8.15.1
+
+    run:
+        - hdf5 1.8.15.1
```

Comment:
We cannot remove the `hdf5` pin. We need `hdf5 1.8.15*` instead of `1.8.16*` to be compatible with the `default` channel `libnetcdf `. We will be able to upgrade only when we have our own `libnetcdf` compiled against `hdf5 1.8.16` and we are sure users won't be mixing default channel packages with conda-forge packages. (The latter we can never really know, but we can provide the majority of the packages compiled against our `hdf5`).

PS: What @pelson meant is to pin to `1.8.15*` instead of the exact version of `1.8.15.1` to avoid the maintenance burden in case new versions on the `1.8.15.x` series pop up.